### DISCUSSION TITLE
How can I access the logged parameters from a registered model?

### DISCUSSION THREAD

**Discussion by MartinRoth** [mlflow/mlflow#6424] [Q&A]
Hi,

how can I assess the logged parameters for a registered model after downloading the model from the registry?
In the metadata I can see the signature and many other features but not the parameters or the metrics.

Any help would be appreciated!

Cheers,
Martin

**Comment by khojarohan4** [comment]
Not sure if this is what you are looking for but you can get the parameters from meta data logged for each experiment

mlflow.get_experiment_by_name('exp_name')
mlflow.search_runs(experiment_ids='exp_id',order_by=['metrics.avgprecision'],max_results=1)

This returns a df and has a column where you could find your logged parameters

**Reply by MartinRoth** [reply]
Thank you very much for the hint! 

Thanks to it I found the `MlflowClient().get_run()` method which actually solves my problem. :)
